webdriver/tests/bidi/external/speculation/conftest.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from typing import Any, Mapping
+
+from webdriver.bidi.modules.script import ContextTarget
+
+
+@pytest.fixture
+def add_speculation_rules(bidi_session):
+    """Helper for adding speculation rules to a page."""
+    
+    async def add_speculation_rules(context: Mapping[str, Any], rules: str):
+        """Add speculation rules script to the page."""
+        await bidi_session.script.evaluate(
+            expression=f"""
+                const script = document.createElement('script');
+                script.type = 'speculationrules';
+                script.textContent = `{rules}`;

```

Comment:
NIT: any particular reason the "`" instead of "'" was used here and in other method?